superset/assets/src/visualizations/presets/CommonChartPreset.js

```diff
@@ -17,21 +17,19 @@
  * under the License.
  */
 import { Preset } from '@superset-ui/core';
```

Comment:
(senv) [root@blade02 presets]# pwd
/root/incubator-superset/superset/assets/src/visualizations/presets
(senv) [root@blade02 presets]# ll
total 8
-rw-r--r--. 1 root root 6453 Feb  4 16:33 MainPreset.js

Hello, there is no such a file in this directory

Note: superset version 0.35.2